base-notebook/Dockerfile

```diff
@@ -52,9 +52,8 @@ RUN useradd -m -s /bin/bash -N -u $NB_UID $NB_USER && \
 
 USER $NB_USER
 
-# Setup jovyan home directory
-RUN mkdir /home/$NB_USER/work && \
-    echo "cacert=/etc/ssl/certs/ca-certificates.crt" > /home/$NB_USER/.curlrc
+# Setup work directory for backward-compatibility
+RUN mkdir /home/$NB_USER/work
```

Comment:
`~/work` is configured as the default `notebook-dir` both so that it could be mounted without disturbing all of the files in the home directory and to avoid cluttering the notebook dashboard page.